oremda/typing/__init__.py

```diff
@@ -72,6 +72,7 @@ class OperateTaskMessage(TaskMessage):
     data_inputs: Dict[PortKey, str] = {}
     meta_inputs: Dict[PortKey, MetaType] = {}
     params: JSONType = {}
+    output_queue: str = ""
```

Comment:
I think the `output_queue` should be strictly required

Reply:
I agree. But the one case I could think of where it might not be required is for a terminate message.

Reply:
The `TerminateTaskMessage` is a separate model though, and shouldn't even have the `output_queue` field

Reply:
Ah yes, I see. I'll make it required here.